fastlane/lib/fastlane/cli_tools_distributor.rb

```diff
@@ -18,8 +18,8 @@ def take_off
         require "fastlane" # this might take a long time if there is no Gemfile :(
 
         # We want to avoid printing output other than the version number if we are running `fastlane -v`
-        if Time.now - before_import_time > 3 && !running_version_command?
-          print_slow_fastlane_warning
+        unless running_version_command?
```

Comment:
I'm not sure I understand this change, if a user has _fastlane_ installed in their gemset, and in their bundle with different versions, not using `bundle exec` will print a different version than will be executed when running with `bundle exec` right?